ci-operator/step-registry/ipi/conf/vsphere/template/ipi-conf-vsphere-template-commands.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+declare vsphere_datacenter
+declare vsphere_datastore
+declare vsphere_url
+declare vsphere_cluster
+source "${SHARED_DIR}/vsphere_context.sh"
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+
+function getVersion() {
+  local release_image=""
+  if [ -n "${RELEASE_IMAGE_INITIAL-}" ]; then
+    release_image=${RELEASE_IMAGE_INITIAL}
+  elif [ -n "${RELEASE_IMAGE_LATEST-}" ]; then
+    release_image=${RELEASE_IMAGE_LATEST}     
+  fi
+  
+  local version=""
+  if [ ${release_image} != "" ]; then
+    oc registry login
+    version=$(oc adm release info ${release_image} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)    
+  fi
+  echo "${version}"
+}
+
```

Comment:
Per the [log](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/41996/rehearse-41996-periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-vsphere-ipi-template-f14/1690901690858868736), try to add the following lines for fixing it:
```
export HOME="${HOME:-/tmp/home}"
export XDG_RUNTIME_DIR="${HOME}/run"
export REGISTRY_AUTH_PREFERENCE=podman # TODO: remove later, used for migrating oc from docker to podman
mkdir -p "${XDG_RUNTIME_DIR}"
```

Reply:
all the occurrence of `oc registry login` need these lines, so I think `ipi-conf-vsphere` also need them.

Reply:
also need the change in ipi-conf-vsphere.